### ISSUE TITLE
Add options to include/remove ligands

### ISSUE THREAD

**Issue by sillitoe** [UCLOrengoGroup/cath-tools#3]
It would be useful if the user could specify which ligands to include/exclude in the superposed structure. Perhaps this could start as defaulting to something sensible, e.g. include any biologically relevant ligands that are close to the structural region of interest (see #1), then adding the ability to modify those selections later on.

**Comment by tonyelewis** [comment]
See #21 for a discussion of how to work-around this issue until it's resolved.

**Comment by tonyelewis** [comment]
A simple version of this has been implemented in 9fb416419dadeb91a7de0776207100e6a79c2ee9, which adds these options to `cath-superpose`:

~~~~~
  --show-dna-within-dist <dist> (=4)       Show DNA within <dist>Å of the alignment regions
  --show-organic-within-dist <dist> (=10)  Show organic molecules within <dist>Å of the alignment regions
~~~~~

We may want to make these options more complicated in the future but I propose that any extra complexity should be guided by experience of the current options and what real use cases they aren't adequately handling.
